src/xtgeo/surface/_regsurf_gridding.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 """Do gridding from 3D parameters"""
 
-
 import warnings
+from typing import Callable
```

Comment:
I think with py10+ it should be `from collections.abc import Callable` which will take some getting used to 😄

Reply:
Thanks :) I didn't know